lib/handlers/healthmonitor.js

```diff
@@ -81,11 +89,33 @@ function initFhHealth() {
     fhhealth.addCriticalTest('Check fh-messaging running', checkFhMessaging);
   } else {
     fhhealth.addCriticalTest('Check fh-ditch status', checkDitch);
+    fhhealth.addCriticalTest('Check apps reachable', checkApps);
   }
 
   return fhhealth;
 }
 
+function checkApps(callback) {
+
+  //dfc.apps.list
+  dfc.dynos.list('detailed', function(err, dynos){
+    console.log(dynos, 'err:', err);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)